### DISCUSSION TITLE
Bootstrap failed to complete: failed waiting for Kubernetes API: Get \"https://api.oc-cluster1.ocamigoes.in:6443/version?timeout=32s\": dial tcp: lookup api.oc-cluster1.ocamigoes.in on 192.168.29.123:53: no such host"  Can anyone help me in fixing this issue.

### DISCUSSION THREAD

**Discussion by gbalamk** [okd-project/okd#730] [Q&A]
Trying to install the OKD 4.7 latest release in AWS Clusters from Centos 8 machine and getting the below error.

Bootstrap failed to complete: failed waiting for Kubernetes API: Get \"https://api.oc-cluster1.ocamigoes.in:6443/version?timeout=32s\": dial tcp: lookup api.oc-cluster1.ocamigoes.in on 192.168.29.123:53: no such host"

Can anyone help me in fixing this issue.
[log-bundle-20210629205959.tar.gz](https://github.com/openshift/okd/files/6742011/log-bundle-20210629205959.tar.gz)

**Comment by vrutkovs** [comment]
`Jun 30 00:57:32 ip-10-0-14-95 bootkube.sh[4169]: Tearing down temporary bootstrap control plane...`

You must remove bootstrap node after `wait-for bootstrap-complete` is done for installation to proceed

**Comment by gbalamk** [comment]
Thanks @vrutkovs , Should i need to shutdown the bootstarp node in AWS ?

**Reply by vrutkovs** [reply]
Yeah, its safe to be terminated and removed from LBs

**Reply by gbalamk** [reply]
I tried shutting down the bootstrap node in AWS but still i am getting the same error , ERROR Attempted to gather ClusterOperator status after installation failure: listing ClusterOperator objects: Get "https://api.oc-cluster.ocamigoes.in:6443/apis/config.openshift.io/v1/clusteroperators": dial tcp: lookup api.oc-cluster.ocamigoes.in on 192.168.29.123:53: no such host

**Reply by duritong** [reply]
No such host means that dns resolution is not working. Either your hosts are getting entries in resolv.conf (through dhcp) that do not resolve your domain or your dns entries were not created.
